### ISSUE TITLE
Put Together Presentation for the Blinded Veterans Association

### ISSUE THREAD

**Issue by brea11y** [department-of-veterans-affairs/va-mobile-app#7293]
Put together a presentation for the November 28th session of the Blinded Veterans Association (BVA) Ambassador Continuing Education Seminar Series. The BVA has provided an agenda that they would like us to talk through. Brea will setup the presentation and should include Jen, Matt, and Tim W. for feedback.

The email / announcement about the presentation that was sent to BVA ambassadors (including the list of topics to cover) is included below, for reference.

-----

The Blinded Veterans Association (BVA) invites you to join us for our BVA Ambassador Continuing Education Seminar Series. The next seminar will be on Tuesday, November 28, 2023, at 2:00 PM Eastern. This session features Brea Blackwelder, Senior Accessibility Designer, Ad Hoc, presenting on the VA Health and Benefits app.

BVA has forged a strong partnership with the VA's Office of Compliance, Risk, and Remediation and Digital Services teams who oversee the development, accessibility, and usability of VA's digital communications tools. Brea will provide an overview of the VA Health and Benefits app, review the app's features, and demonstrate how blind and low vision veterans may use the app.

The seminar will be presented by Brea from the Ad Hoc team and will cover the following topics:
- What is the VA's Health and Benefits app?
- Review of the tools and features of the VA Health and Benefits app
- How blind and low vision veterans may use the app
